### PR TITLE
Include lock component in taoItems

### PR DESCRIPTION
**Pull request by jbout**

### CODE REVIEW

**Comment by llecaque** [comment]
@krampstudio can you have a quick look to js and tpl

File: views/js/controller/items/edit.js [hunk 1]
```diff
@@ -17,7 +18,29 @@ define(['module', 'layout/actions'], function(module, actions){
                 if(authoringAction){
                     authoringAction.state.disabled = !config.isAuthoringEnabled;
                 }
-                actions.updateState(); 
+                actions.updateState();
+                
+                if(config.msg !== false){
+                    var lk = lock($('#lock-box')).hasLock(config.msg,
```
Comment (krampstudio) [inline views/js/controller/items/edit.js]: `$('#lock-box')` is used twice. please add a variable

**Comment by krampstudio** [comment]
Is it a test implementation ? ortherwise removev the `console.log` ...

**Comment by krampstudio** [comment]
I've checked the code and tested if there are not regression without ACLs. I'll merge it and let @jbout do further testing if needed.

**Pull request merged**